clover.rb

```diff
@@ -477,6 +513,22 @@ def assets
     if api?
       response.json = true
       response.skip_content_security_policy!
+
+      r.rodauth
+      rodauth.check_active_session
+      rodauth.require_authentication
+
+      # Validate request against OpenAPI schema
+      begin
+        schema_validator = SCHEMA_ROUTER.build_schema_validator(r)
+        schema_validator.request_validate(r)
+
+        raise Committee::NotFound, "That request method and path combination isn't defined." if !schema_validator.link_exist?
+      rescue JSON::ParserError => e
+        raise Committee::InvalidRequest.new("Request body wasn't valid JSON.", original_error: e)
```

Comment:
Do we have to use `original_error` here?  Is Exception#cause not sufficient for this?

Reply:
If I recall correctly, `original_error` is a reflection of how committee does some of this internally. Open to alternative ways to do it.

Reply:
If that's how committee already works, it's fine to keep using it.  Sequel uses something similar, since that type of nested exception handling was supported before Exception#cause was added to Ruby.  I guess when committee was first being developed, it had to support Ruby versions that didn't have Exception#cause (it was added in Ruby 2.1).